packages/static-wado-webserver/lib/index.mjs

```diff
@@ -92,6 +95,10 @@ const methods = {
     await addQueryCall(router, "/studies", params, "studyQuery");
     router.get("/studies", qidoMap);
     router.get("/studies/:studyUID/series", qidoMap);
+
```

Comment:
Why don't you move the entire addDicomWeb call into setRoutes?  That way it is actually obvious what you are doing, rather than have a tiny function to do just a bit of the work?

Reply:
This should be the ideal. I didnt do because you have opened PR which might cause more overwork to you.

Reply:
Add a commit for commit#:  dbf64489554aa206cb440a6c48688be9c49747ce

just for reference in case we want to drop the changes.